src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c

```diff
@@ -1,32 +1,26 @@
-/*
+/**
+ * SPDX-License-Identifier: Apache-2.0
  * Copyright (C) 2016-2022 Xilinx, Inc. All rights reserved.
+ * Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
  *
  * Authors: Lizhi.Hou@xilinx.com
- *
- * This software is licensed under the terms of the GNU General Public
- * License version 2, as published by the Free Software Foundation, and
- * may be copied, distributed, and modified under those terms.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
  */
 
-#include <linux/pci.h>
-#include <linux/kernel.h>
 #include <linux/aer.h>
-#include <linux/version.h>
-#include <linux/module.h>
-#include <linux/pci.h>
 #include <linux/crc32c.h>
-#include <linux/random.h>
 #include <linux/iommu.h>
+#include <linux/kernel.h>
+#include <linux/module.h>
 #include <linux/pagemap.h>
-#include "../xocl_drv.h"
-#include "xocl_errors.h"
+#include <linux/pci.h>
+#include <linux/random.h>
+#include <linux/version.h>
+
 #include "common.h"
 #include "version.h"
```

Comment:
It might be helpful if a comment was added to indicate which version.h is being used (since there are multiple ones being included).